users/config.rst

```diff
@@ -64,7 +64,14 @@ The database contains the following files:
 Config File Format
 ------------------
 
-The following shows an example of the default configuration file (IDs will differ):
+The following shows an example of a default configuration file (IDs will differ):
+
+
+.. note::
+   The config examples are present for illustration. Do **not** copy them
+   entirely to use as your config. They are likely out-of-date and the values
+   might not correspond any more to the defaults.
```

Comment:
The "any more" part feels grammatically incorrect and not sure if it adds any value (if its out of date, its out of date).

Reply:
How about "the values may no longer correspond to the defaults"?